src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+* On Exception, Activity Status is set to Error. The Exception Message
```

Comment:
Removed the Activity Status Description that was being set during unhandled exceptions. Activity Status will continue to be reported as `Error`